hosted/eks/k8s_chart_support/k8s_chart_support_suite_test.go

```diff
@@ -5,6 +5,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/onsi/ginkgo/v2"
```

Comment:
```suggestion

```

Reply:
This shouldn't be needed once you replace `ginko.GinkgoLogr.Info` by `GinkgoLogr.Info` later.